abstractions/model_free/sac/train.py

```diff
@@ -140,7 +142,7 @@ def episode_loop(env, test_env, agent, replay_buffer, args, writer):
     device = torch.device('cpu')
 
 # Initialize model
-agent = SAC(env.observation_space.shape[0], env.action_space, device, args)
+agent = SAC(env.observation_space.shape, env.action_space, device, args)
```

Comment:
Will this cause a problem for the MLP version?

Reply:
Nope, because I've adapted the MLP version to handle it.